app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -138,18 +156,31 @@ class Contact extends React.Component {
           secondary_telephone: yup.string().phone().max(200, 'Max length exceeded, please limit to 200 characters.'),
           primary_telephone_type: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.'),
           secondary_telephone_type: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.'),
-          international_telephone: yup.string().max(50, 'Max length exceeded, please limit to 50 characters.').nullable(),
+          international_telephone: yup.string().max(50, 'Max length exceeded, please limit to 50 characters.'),
           email: yup.string().email('Please enter a valid Email.').max(200, 'Max length exceeded, please limit to 200 characters.'),
           confirm_email: yup.string().oneOf([yup.ref('email'), null], 'Confirm Email must match.'),
           preferred_contact_method: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.'),
+          alternate_contact_type: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.'),
+          alternate_contact_name: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.'),
+          alternate_primary_telephone: yup.string().phone().max(200, 'Max length exceeded, please limit to 200 characters.'),
+          alternate_secondary_telephone: yup.string().phone().max(200, 'Max length exceeded, please limit to 200 characters.'),
```

Comment:
My comment isn't about these lines, but with `alternate_primary_telephone` and `alternate_secondary_telephone`, I think we should be giving them the same treatment as `primary_telephone` and `secondary_telephone`. Right now we're storing values in the database like `555-444-3333` for alternate and `+15554443333` for primary.

As far as I can tell the necessary changes would include stripping hyphens here:
https://github.com/SaraAlert/SaraAlert/blob/95ecb9c20c41b22682d62a08cc2aa8d3ee98d8a0/app/javascript/components/enrollment/steps/Contact.js#L74-L75
and converting to E164 format here:
https://github.com/SaraAlert/SaraAlert/blob/95ecb9c20c41b22682d62a08cc2aa8d3ee98d8a0/app/javascript/components/enrollment/Enrollment.js#L106-L111
This way we will have more consistency of how we're storing our phone numbers in the database (including the `CloseContact.primary_telephone`).

Also I think the way it is now is causing a bug where `alternate_primary_telephone` and `alternate_secondary_telephone` aren't being exported.